### DISCUSSION TITLE
css files and pictures not rendering

### DISCUSSION THREAD

**Discussion by adighiero1** [vercel/community#7378] [Help] [answered]
### Summary

I recently built a static website for a friend of mine and asked me to use vercel to deploy it. When I did the webiste looks like this:https://links-eight-nu.vercel.app/ here is my github: https://github.com/adighiero1/links

I've tried redeploying the website. Checking the logs but it doesn't list any errors.

### Steps to Reproduce

you can clone repo if you want. the link for the webapp is provided above as well.

**Comment by ogbotemi-2000** [comment, marked as answer]
Hello, I saw this discussion unanswered and I felt I'd help.
For static websites, which I believe yours is one, toggle off the switch for the input that defaults to `public` as output folder in your build settings. Leave the input field blank.

The website link displays so because Vercel is adding a folder `/public/` to the path to your used external assets, CSS, JS, images. Since no such file or folder exists, you get an empty response with no results

**Reply by pawlean** [reply]
Thanks for sharing this @ogbotemi-2000 and helping out. This is also what my answer would be! Did that help, @adighiero1?

**Reply by adighiero1** [reply]
yes. thank you very much

[a comment by an automated account is not shown]
